src/components/PeopleTable.tsx

```diff
@@ -32,9 +112,14 @@ export const PeopleTable = () => {
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Born
-              <a href="#/people?sort=born&amp;order=desc">
+              <a
+                href="#/people?sort=born&amp;order=desc"
```

Comment:
The `href` attribute in the 'Born' column header includes `&amp;order=desc`, which might not be necessary since the `handleSortByCategory` function manages the order state. Consider removing it to avoid potential confusion.